src/components/top_bar/top_bar.ts

```diff
@@ -366,8 +367,13 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
     setFormatter(this.env, value);
   }
 
-  toggleAlign(align: Align) {
-    setStyle(this.env, { ["align"]: align });
+  toggleHorizontalAlign(align: Align) {
```

Comment:
I think we should create a task to rename `Align` to `HorizontalAlign` to be more consistent. But it would require a migration...
What do you think ?

Reply:
Task created